src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,134 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.exceptions.PasswordException;
+import core.basesyntax.service.exceptions.UserExistException;
+import core.basesyntax.service.exceptions.UserNullException;
+import core.basesyntax.service.exceptions.UserYoungException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+
+    private static final int MIN_USER_AGE = 18;
+    private static final String VALID_PASSWORD = "123456";
+    private static final String DEFAULT_LOGIN = "login";
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @AfterEach
+    void clearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_normalUser_ok() {
+        User normalUser = createUser(DEFAULT_LOGIN, VALID_PASSWORD, MIN_USER_AGE);
+        User registeredUser = registrationService.register(normalUser);
+
+        Assertions.assertSame(normalUser, registeredUser,
+                "register method must return same user as was supplied");
+        assertNotNull(normalUser.getId(), "User must have id after registration");
+        // check db state
+        List<User> expectedDbState = new ArrayList<>();
+        expectedDbState.add(normalUser);
+        assertEquals(Storage.people, expectedDbState, "User must be saved in db");
```

Comment:
in method assertEquals() in the first place - the expected value (valid value), in the second place - actual result (the result that was got and must be tasted)
fix the problem with variable names in all tests 
```suggestion
        assertEquals(Storage.people, actualDbState, "User must be saved in db");
```